modules/contacts/modules.php

```diff
@@ -10,6 +10,70 @@
 
 require APP_PATH.'modules/contacts/hm-contacts.php';
 
+/**
+ * @subpackage contacts/handler
+ */
+class Hm_Handler_process_contact_auto_collect_setting extends Hm_Handler_Module {
+    public function process() {
+        function contact_auto_collect_callback($val) {
+            return $val;
+        }
+        process_site_setting('contact_auto_collect', $this, 'contact_auto_collect_callback', false, true);
+    }
+}
+
+/**
+ * @subpackage contacts/handler
+ */
+class Hm_Handler_store_contact_message extends Hm_Handler_Module {
+    public function process() {
+        if ($this->get('collect_contacts', false)) {
+            $addresses = process_address_fld($this->request->post['compose_to']);
+            $contact_list = $this->user_config->get('contacts', []);
```

Comment:
Latest master has moved to a Repository pattern for storing and retrieving contacts. Please check how it is done and use appropriate code to get contacts - they are already initialized in one of the handler methods.